backend/app/services/llm/jobs.py

```
@@ -347,43 +375,56 @@ def execute_job(
-                    ]
+                    if safe_output.get("bypassed"):
+                        logger.info(
+                            "[execute_job] Guardrails bypassed (service unavailable)"
+                        )
+
+                    elif safe_output["success"]:
+                        response.response.output.content.value = safe_output["data"][
+                            "safe_text"
+                        ]
+
+                        if safe_output["data"]["rephrase_needed"] == True:
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `safe_output["data"]["rephrase_needed"]:` for truth checks

Replace with `safe_output["data"]["rephrase_needed"]`

(E712)